File: cs/Iris/ParserUtils/LexParseException/InvalidLeftTypeOperand.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Iris.ParserUtils.LexParseException
{
    public class InvalidLeftTypeOperand : InvalidTypeOperand
    {
        public InvalidLeftTypeOperand(int lNum, int cNum, Token.TokenValue token)
            : base(lNum, cNum, token)
        {
            
        }

        public override string ExceptionToString()
        {
            return base.ExceptionToString("left");
        }
    }
}

File: java/Iris/src/com/rokuan/iris/exception/InvalidLeftTypeOperand.java
package com.rokuan.iris.exception;

import com.rokuan.iris.parser.Token;

public class InvalidLeftTypeOperand extends InvalidTypeOperand
{
	public InvalidLeftTypeOperand(int lNum, int cNum, Token.TokenValue token)
	{
		super(lNum, cNum, token);
	}

	public String ExceptionToString()
	{
		return super.ExceptionToString("left");
	}
}
